lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -16,16 +16,10 @@ class CheckoutV2Gateway < Gateway
       LIVE_ACCESS_TOKEN_URL = 'https://access.checkout.com/connect/token'
       TEST_ACCESS_TOKEN_URL = 'https://access.sandbox.checkout.com/connect/token'
 
-      def initialize(options = {})
-        @options = options
-        @access_token = options[:access_token] || nil
+      BASIC_SECRET_KEY_FORMAT = /^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i
```

Comment:
Did you get this format from their docs?

Reply:
No, this is an assumption that I have, I am waiting for confirmation from the Checkout.com team, I am going to ask if there is any progress on this topic

Reply:
Okay, then let's waiting on this until we hear back. I'll move the ticket to waiting

Reply:
This format will no longer be necessary, I did some tests and now Checkout supports both formats with the 'Bearer' form